src/components/PeopleList.tsx/PeopleList.tsx

```diff
@@ -0,0 +1,30 @@
+import { Person } from '../../types';
+import { PersonInfo } from '../PersonInfo';
+
+interface Props {
+  people: Person[];
+}
+
+export const PeopleList: React.FC<Props> = ({ people }) => (
+  <table
+    data-cy="peopleTable"
+    className="table is-striped is-hoverable is-narrow is-fullwidth"
+  >
+    <thead>
+      <tr>
+        <th>Name</th>
+        <th>Sex</th>
+        <th>Born</th>
+        <th>Died</th>
+        <th>Mother</th>
+        <th>Father</th>
```

Comment:
```suggestion
        <th>Name</th>
        
        <th>Sex</th>
        
        <th>Born</th>
        
        <th>Died</th>
        
        <th>Mother</th>
        
        <th>Father</th>
```

Reply:
I think we need separate this homogeneous parts